### PR TITLE
[load] high scale test check | download-files update console message

### PR DESCRIPTION
**Pull request by mbhardwaj-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/load/azext_load/data_plane/load_test_run/custom.py [hunk 1]
```diff
@@ -157,14 +158,19 @@ def _download_results_file(test_run_output_artifacts, test_run_id, path):
             logger.warning("Results file downloaded to %s", file_path)
         else:
             logger.info("No results file found for test run %s", test_run_id)
+    elif _is_high_scale_test_run(test_run_data):
+        logger.warning(
+            "High scale test run %s results file is not available for download",
```
Comment (mbhardwaj-msft) [inline src/load/azext_load/data_plane/load_test_run/custom.py:163]: @saloni to review

File: src/load/azext_load/data_plane/load_test_run/custom.py [hunk 2]
```diff
@@ -173,14 +179,19 @@ def _download_reports_file(test_run_output_artifacts, test_run_id, path):
             logger.warning("Report file downloaded to %s", file_path)
         else:
             logger.info("No report file found for test run %s", test_run_id)
+    elif _is_high_scale_test_run(test_run_data):
+        logger.warning(
+            "High scale test run %s report file is not available for download",
```
Comment (mbhardwaj-msft) [inline src/load/azext_load/data_plane/load_test_run/custom.py:184]: @saloniagrawal1997 to review

File: src/load/azext_load/data_plane/load_test_run/custom.py [hunk 3]
```diff
@@ -189,6 +200,11 @@ def _download_logs_file(test_run_output_artifacts, test_run_id, path):
             logger.warning("Log file downloaded to %s", file_path)
         else:
             logger.info("No log file found for test run %s", test_run_id)
+    elif _is_high_scale_test_run(test_run_data):
+        logger.warning(
+            "High scale test run %s log file is not available for download",
```
Comment (mbhardwaj-msft) [inline src/load/azext_load/data_plane/load_test_run/custom.py:205]: @saloniagrawal1997 to review

File: src/load/azext_load/data_plane/load_test_run/custom.py [hunk 4]
```diff
@@ -136,12 +137,14 @@ def stop_test_run(cmd, load_test_resource, test_run_id, resource_group_name=None
 def copy_test_run_artifacts_url(cmd, load_test_resource, test_run_id, resource_group_name=None):
     client = get_testrun_data_plane_client(cmd, load_test_resource, resource_group_name)
     logger.info("Fetching test run copy artifacts SAS URL for test run %s", test_run_id)
+    logger.warning("You can use the SAS URL in the Azure Storage Explorer or AzCopy "
```
Comment (mbhardwaj-msft) [inline src/load/azext_load/data_plane/load_test_run/custom.py]: @saloniagrawal1997 to review

File: src/load/azext_load/data_plane/load_test_run/custom.py [hunk 5]
```diff
@@ -236,15 +250,28 @@ def download_test_run_files(
     if test_run_input:
         _download_input_file(test_run_input_artifacts, test_run_id, path)
 
+    is_high_scale_test_run = _is_high_scale_test_run(test_run_data)
+    high_scale_test_run_message = ""
     if test_run_log:
-        _download_logs_file(test_run_output_artifacts, test_run_id, path)
+        if is_high_scale_test_run:
+            high_scale_test_run_message += f"High scale test run {test_run_id} log file " \
+                "is not available for download. "
```
Comment (mbhardwaj-msft) [inline src/load/azext_load/data_plane/load_test_run/custom.py:257]: @saloniagrawal1997 to review

File: src/load/azext_load/data_plane/load_test_run/custom.py [hunk 6]
```diff
@@ -236,15 +250,28 @@ def download_test_run_files(
...
+        else:
+            _download_logs_file(test_run_output_artifacts, test_run_id, path)
 
     if test_run_results:
-        _download_results_file(test_run_output_artifacts, test_run_id, path)
+        if is_high_scale_test_run:
+            high_scale_test_run_message += f"High scale test run {test_run_id} results file " \
+                "is not available for download. "
```
Comment (mbhardwaj-msft) [inline src/load/azext_load/data_plane/load_test_run/custom.py:264]: @saloniagrawal1997 to review

File: src/load/azext_load/data_plane/load_test_run/custom.py (hunk 4 above)
Comment (saloniagrawal1997) [reply src/load/azext_load/data_plane/load_test_run/custom.py]: You can use the SAS URL with Azure Storage Explorer or AzCopy to access the storage resource.

File: src/load/azext_load/data_plane/load_test_run/custom.py (hunk 5 above)
Comment (saloniagrawal1997) [reply src/load/azext_load/data_plane/load_test_run/custom.py:257]: Log file for high-scale test {test_run_id} is not available for download. Use the 'get-artifacts-url' command to fetch the SAS URL and access the file.

File: src/load/azext_load/data_plane/load_test_run/custom.py (hunk 6 above)
Comment (saloniagrawal1997) [reply src/load/azext_load/data_plane/load_test_run/custom.py:264]: Result files for high-scale test {test_run_id} is not available for download. Use the 'get-artifacts-url' command to fetch the SAS URL and access the files.

**Review by prativen — APPROVED** [review]

**Pull request merged**